docs/draft/howto/single-ownnamespace-install.md

```diff
@@ -0,0 +1,105 @@
+## Description
+
+!!! note
+This feature is still in *alpha* the `SingleOwnNamespaceInstallSupport` feature-gate must be enabled to make use of it.
+See the instructions below on how to enable it.
+
+---
+
+A component of OLMv0's multi-tenancy feature is its support of four [*installModes*](https://olm.operatorframework.io/docs/advanced-tasks/operator-scoping-with-operatorgroups/#targetnamespaces-and-their-relationship-to-installmodes):
+for operator installation:
+
+ - *OwnNamespace*: If supported, the operator can be configured to watch for events in the namespace it is deployed in.
+ - *SingleNamespace*: If supported, the operator can be configured to watch for events in a single namespace that the operator is not deployed in.
```

Comment:
I was wondering if we need to support this one.
Since this is the MultiNamespace case behind the scenes, the config here will watch its own namespace plus one extra namespace.
Or did I miss something?

Reply:
In `SingleNamespace` mode that operator will still only watch for events in a single namespace (not its own namespace + another). I.e. it's "configured to watch for events in a _single_ namespace that the operator is **not** deployed in. While under the hood the watch namespace parameter is indeed a `[]string`:

If len(watch_namespace) == 0 then AllNamespaces
If len(watch_namespace) > 1 then MultiNamespaces
If len(watch_namespace) == 1 && watch_namespace[0] == install namespace then OwnNamespace
If len(watch_namespace) == 1 && watch_namespace[0] != install namespace then SingleNamespace

Reply:
Thank you for the reminder!

I was trying to recall whether the SingleNamespace mode in OLM would result in the operator watching both the installed namespace and the explicitly informed namespace to ensure the operator itself functions properly.

However, as discussed, OLM is responsible for deploying and managing the operator, so the operator does not need to watch its own namespace. Instead, it only watches the namespace specified in the OperatorGroup, and OLM ensures the operator itself is running correctly.

This clarifies that in SingleNamespace mode, the operator will watch only the informed namespace, and the installed namespace is not included unless explicitly configured.

We can close this one